classes/core/Application.inc.php

```diff
@@ -118,6 +118,7 @@ public function getDAOMap()
             'AuthorDAO' => 'APP\article\AuthorDAO',
             'IndividualSubscriptionDAO' => 'APP\subscription\IndividualSubscriptionDAO',
             'InstitutionalSubscriptionDAO' => 'APP\subscription\InstitutionalSubscriptionDAO',
+            // TODO: What should be done with this?
             'IssueDAO' => 'APP\issue\IssueDAO',
```

Comment:
I'm not sure what the usage here was. Can this be remove or should it point to the new DAO?